fix_includes.py

```diff
@@ -1929,7 +1929,8 @@ def FixFileLines(iwyu_record, file_lines, flags):
   # For every move-span in our file -- that's every #include and
   # forward-declare we saw -- 'decorate' the move-range to allow us
   # to sort them.
-  move_spans = set([fl.move_span for fl in file_lines if fl.move_span])
+  seen = set()
+  move_spans = [fl.move_span for fl in file_lines if fl.move_span and not (fl.move_span in seen or seen.add(fl.move_span))]
```

Comment:
You're changing `move_spans` from a set to a list to preserve order, right? I wonder if you could use `OrderedSet`, defined earlier in fix_includes.py, instead?

    move_spans = OrderedSet([fl.move_span for fl in file_lines if fl.move_span])

Reply:
Yes, the intention is to retain the order from `file_lines`.  This is the first time I've written anything in python, I'll yield to your judgement if you think it's appropriate.